sidebars.js

```diff
@@ -33,6 +33,25 @@ module.exports = {
 					label: 'SEO tags',
 					items: [
 
+                        {
+                            type: 'category',
+                            label: 'Titles',
+                            items: [
+                                'features/seo-tags/titles/overview',
+                                'features/seo-tags/titles/functional-specification',
+                                'features/seo-tags/titles/api',
+                            ],
+                        },
+
+                        {
+							type: 'category',
+							label: 'Descriptions',
+							items: [
+								'features/seo-tags/descriptions/overview',
+								'features/seo-tags/descriptions/functional-specification',
+								'features/seo-tags/descriptions/api',
+							],
```

Comment:
Indentation seems off.

Reply:
Can you wrangle this from your side? Think it's a Windows thing; struggling to align!

Reply:
Tabs vs spaces :P